routes/index.js

```diff
@@ -0,0 +1,10 @@
+var express = require('express');
```

Comment:
Next story: typescript

Reply:
Created Issue for converting to typescript.